src/app/component/send-audio/send-audio.component.ts

```diff
@@ -0,0 +1,21 @@
+import { ChangeDetectionStrategy, Component, OnInit } from "@angular/core";
+import { List } from "immutable";
+import { Page } from "src/app/helpers/page/pageDecorator";
+import { sendAudioCategory, sendAudioMenuItem } from "./send-audio.menus";
+
+@Page({
+  category: sendAudioCategory,
+  menus: {
+    actions: List(),
+    links: List()
+  },
+  self: sendAudioMenuItem
+})
+@Component({
+  selector: "app-send-audio",
+  template: `
+    <app-cms page="sendAudio.html"></app-cms>
```

Comment:
pull the URL out from somewhere in code, please. The component instance is fine for now